apps/dashboard/app/javascript/path_selector/path_selector_data_table.js

```diff
@@ -161,7 +160,8 @@ export class PathSelectorTable {
   }
 
   storageKey() {
-    return `${this.tableId}_last_visited`;
+    const underscore_path = window.location.pathname.replaceAll('/', '_');
+    return `${this.tableId}${underscore_path}_last_visited`;
```

Comment:
This doesn't actually fix this issue - but it can help with uniqueness in that folks can reuse the same name in multiple forms. So this ensures the cache key is unique to that form.